driver-http/src/main/resources/activities/documents-api/http-docsapi-search-advanced.yaml

```diff
@@ -0,0 +1,315 @@
+              },
+              "secondary":  {}
+            },
+            "coordinates": [
+                            {lat},
+                            {lng}
+            ],
+            "children":     [],
+            "friends": [
+                            "{friend_id}"
+            ],
+            "debt":         null,
+            "match1":       {match1},
+            "match2":       "{match2}",
+            "match3":       {match3}
+            {additional_fields}
+          }
+        tags:
+          name: rampup-insert
+  
+  - name: rampup-in
+    tags:
+      phase: rampup-read
+      filter: in
+    statements:
+      - rampup-in: GET <<protocol:http>>://{weighted_hosts}:<<stargate_port:8082>><<path_prefix:>>/v2/namespaces/<<keyspace:docs_search_advanced>>/collections/<<table:docs_collection>>?where=%7B%22match1%22%3A%7B%22%24in%22%3A%5B0%5D%7D%7D&page-size=<<page-size,3>>&fields=<<fields,%5b%5d>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        tags:
+          name: rampup-in
+  
+  - name: rampup-not-in
+    tags:
+      phase: rampup-read
+      filter: not-in
+    statements:
+      - rampup-not-in: GET <<protocol:http>>://{weighted_hosts}:<<stargate_port:8082>><<path_prefix:>>/v2/namespaces/<<keyspace:docs_search_advanced>>/collections/<<table:docs_collection>>?where=%7B%22match2%22%3A%7B%22%24nin%22%3A%5B%22false%22%5D%7D%7D&page-size=<<page-size,3>>&fields=<<fields,%5b%5d>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        tags:
+          name: rampup-not-in
+
+  - name: rampup-mem-and
+    tags:
+      phase:  rampup-read
+      filter: mem-and
+    statements:
+      - rampup-mem-and: GET <<protocol:http>>://{weighted_hosts}:<<stargate_port:8082>><<path_prefix:>>/v2/namespaces/<<keyspace:docs_search_advanced>>/collections/<<table:docs_collection>>?where=%7B%22match2%22%3A%7B%22%24eq%22%3A%22true%22%7D%2C%22match3%22%3A%7B%22%24ne%22%3A%20false%7D%7D&page-size=<<page-size,3>>&fields=<<fields,%5b%5d>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        tags:
+          name: rampup-mem-and
+
+  - name: rampup-mem-or
+    tags:
+      phase:  rampup-read
+      filter: mem-or
+    statements:
+      - rampup-mem-or: GET <<protocol:http>>://{weighted_hosts}:<<stargate_port:8082>><<path_prefix:>>/v2/namespaces/<<keyspace:docs_search_advanced>>/collections/<<table:docs_collection>>?where=%7B%22%24or%22%3A%5B%7B%22match1%22%3A%7B%22%24lt%22%3A1%7D%7D%2C%7B%22match3%22%3A%7B%22%24exists%22%3Atrue%7D%7D%5D%7D&page-size=<<page-size,3>>&fields=<<fields,%5b%5d>>
```

Comment:
hmm `"$exists":true` is not in-memory one, only `$exists":false`, you can make up a field: `{"not_existing":{"$exists":false}}`..

Or let again specify, there are three types:
* Cassandra ones (lt, gt)
* In memory single row checked ($in, $exists: true)
* In memory whole document ($ne, $nin, $exists: false)

So I am not sure what case you wanted here..

Reply:
I was actually taking from the document we have:
```Search with $or (C*-supported + one that does not evaluate on missing)```

Reply:
If this is the case, I am fine!